### ISSUE TITLE
Numerical Output Types, Dice & Cards Won't Roundtrip

### ISSUE THREAD

**Issue by shannona** [BlockchainCommons/seedtool-cli-rust#3]
Here's an example using `base10`:
```
$ seedtool -i hex 59f2293a5bce7d4de59e71b4207ac5d2 -o base10 | seedtool -i base10
6bed8082369b43155a7231437710c2a3
```
This is also true for `bits`, `base6`, and `ints`. Also `dice` & `cards`.

I'd wondered if this was because they were getting cut off to 32 bytes, which is not the same as the lengths that I tend to be inputting, but no idea. 

I note this was the way "cseedtool" worked too, but I don't understand any purpose of outputting these formats that can't be used to regenerate the seed.

**Comment by shannona** [comment]
Per discussions with @wolfmcnally the main issue here is probably that we don't allow people to input an existing seed and output these forms, which are just intended for use as random input. So that probably means can't match -i of hex/envelope/bip39/btw*/etc with output of bits/base6/base10/ints/dice/card

**Comment by shannona** [comment]
Or really any input, which is how the manual now says it works.

**Comment by wolfmcnally** [comment]
This has been fixed as of 0.1.6.

- The formats `base6`, `base10`, `bits`, `cards`, `dice`, and `ints`, are intended to be used as user-provided input entropy for generating seeds. `seedtool` can use them as input to generate a seed in any of the other formats, or it can generate these directly using the `random` input mode (which you only would need to do when simulating user-generated entropy). However, `seedtool` will now return an error if any of these formats are used for output with anything other than `random` input (the default). This means it will also return an error if asked to convert between these entropy input formats, for example, from `bits` to `dice`, as these are not reversible conversions.
